src/Register.php

```diff
@@ -0,0 +1,45 @@
+<?php namespace Leean\Elements;
+
+use Leean\Elements\Options;
+
+/**
+ * Class to provide predefined ACF elements.
+ */
+class Register
+{
+	/**
+	 * The location for the elements.
+	 *
+	 * @var array
+	 */
+	private static $_location = [];
```

Comment:
It would have to be element/location pairs anyway (otherwise you have to put all the elements in the same location). I would do something like:

````
Register::add( $element, $location )
````

where $location is optional (it could throw an exception if it's not define and there's no default). Then you need to call this function once for each element you want to add.